datacapture/src/main/java/com/google/android/fhir/datacapture/fhirpath/ExpressionEvaluator.kt

```diff
@@ -97,7 +97,8 @@ object ExpressionEvaluator {
     questionnaireItem: QuestionnaireItemComponent,
     questionnaireResponseItem: QuestionnaireResponseItemComponent?,
     expression: Expression,
-    questionnaireItemParentMap: Map<QuestionnaireItemComponent, QuestionnaireItemComponent>
+    questionnaireItemParentMap: Map<QuestionnaireItemComponent, QuestionnaireItemComponent>,
+    launchContextMap: Map<String, Resource>?
```

Comment:
we love trailing commas - so please try to use them in this PR throughout.

```suggestion
    launchContextMap: Map<String, Resource>?,
```

Reply:
okay :D